### DISCUSSION TITLE
Set minimum file size for ingestion

### DISCUSSION THREAD

**Discussion by cfc62** [chuot/rdio-scanner#101] [Ideas]
I'm using rdio-scanner with SDR Trunk with NBFM. As it stands today (and may for some time to come), there's no PL or DCS decode on SDRT. It's power squelch - so fundamentally the audio files generated are different from those coming from P25 systems. Not every file is useful - and I suspect this is the case for AM as well, but haven't tried that just yet. 

What using power squelch effectively means is that there will be a lot of very small files generated for any transient bursts of RF energy - which happen a lot if you're recording dozens of frequencies. There's really nothing of use in these files, it's just a 40-50-100 ms noise burst. They're small - anything under 1k is completely useless while those files between 1k and 2k are _usually_ useless. 

Being able to set a file size threshold for ingestion would be helpful. This would keep the db tidy (it's pretty large even with prune set to 1 day) and keep mp3 to mp4 conversion to a minimum as well. 

I don't know if MP3 tags show duration besides integer seconds, I had thought of that as an alternative but a lot can be said in less than a second and deleting them would be bad, I think.

**Comment by chuot** [comment]
Idealy the recorder shouldn't recorder too short conversation, but that's something doable on my side. I may add some configurable limits in regards to the minimal duration threshold like globaly, per system or per talkgroup.

Thanks for the idea!

**Reply by cfc62** [reply]
Thanks. Agree that the source should not be generating these files, but Denny has been preoccupied with things other than SDRTrunk for a while and I don't think it will get attention for some time. 

Ideally we'd have PL and DCS and this wouldn't be a problem, but here we are :^)
